blocks/header/searchbar.js

```diff
@@ -1,18 +1,8 @@
-import { loadScript } from '../../scripts/aem.js';
-import { getConfigValue } from '../../scripts/configs.js';
+import { render as provider } from '@dropins/storefront-search/render.js';
```

Comment:
https://github.com/adobe-commerce/adobe-demo-store/blob/main/blocks/product-list-page/product-list-page.js#L1

```suggestion
/* eslint-disable import/no-unresolved */
import { render as provider } from '@dropins/storefront-search/render.js';
```